datajoint/autopopulate.py

```diff
@@ -16,28 +17,27 @@ class AutoPopulate(metaclass=abc.ABCMeta):
     """
     AutoPopulate is a mixin class that adds the method populate() to a Relation class.
     Auto-populated relations must inherit from both Relation and AutoPopulate,
-    must define the property populated_from, and must define the callback method _make_tuples.
```

Comment:
A good portion of my computed and imported tables rely on this feature for more efficient computation. Let's not make things complicated by providing several ways of doing the same thing. Calling it `populated_from` was a team decision, it is descriptive, and the ease with which something can be used in discussions should not be a primary reason to choose property names. If most computed tables rely on the default value, why not change it in Matlab?